src/openrct2-ui/windows/Ride.cpp

```diff
@@ -2915,33 +2915,33 @@ static void window_ride_vehicle_paint(rct_window *w, rct_drawpixelinfo *dpi)
 
     // Description
     y += gfx_draw_string_left_wrapped(dpi, &rideEntry->naming.description, x, y, 300, STR_BLACK_STRING, COLOUR_BLACK);
-    y += 5;
+    y += 2;
 
     // Capacity
     gfx_draw_string_left(dpi, STR_CAPACITY, &rideEntry->capacity, COLOUR_BLACK, x, y);
-    y += 5;
 
     // Excitement Factor
     factor = rideEntry->excitement_multiplier;
     if (factor > 0)
     {
-        y += 10;
+        y += 12;
         gfx_draw_string_left(dpi, STR_EXCITEMENT_FACTOR, &factor, COLOUR_BLACK, x, y);
     }
 
     // Intensity Factor
     factor = rideEntry->intensity_multiplier;
     if (factor > 0)
     {
-        y += 10;
+		x += 150;
```

Comment:
Could you apply the x offset in the call to `gfx_draw_string_left`? That way, you don't have to subtract it later on.

Also, and this is very minor, please indent using spaces, not tabs.

Reply:
Alternatively, you could if/else on `font_get_line_height(FONT_SPRITE_BASE_SMALL) == 10` and only apply the change for larger, non-sprite fonts.